sysbuild/Kconfig.netcore

```diff
@@ -126,6 +126,11 @@ config NETCORE_IPC_RADIO_IEEE802154
 
 endif # NETCORE_IPC_RADIO
 
+config NETCORE_CUSTOM_RADIO
```

Comment:
I think it may be cleaner to put this choice extension inside the sample directory.